.github/workflows/doc-tests.yaml

```diff
@@ -2,20 +2,33 @@ name: Lint (Docs)
 run-name: Lint (Docs)
 on:
   pull_request:
-    paths:
-      - 'CHANGELOG.md'
-      - 'docs/**'
-      - 'examples/**'
+
   merge_group:
-    paths:
-      - 'docs/**'
-      - 'examples/**'
-      - 'CHANGELOG.md'
+
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
  changes:
    name: Check for relevant changes
    runs-on: ubuntu-latest
```

This is more legible when scanning the nav.  For example:

![image](https://github.com/gravitational/teleport/assets/187314/25d76c29-b949-4864-9cff-89abcf687f86)

It isn't intuitive what "changes" means above.